src/Orc.Controls/Orc.Controls.Shared/Controls/AnimatedGif/AnimatedGif.cs

```diff
@@ -166,11 +166,19 @@ private Bitmap GetBitmapResourceFromAssembly(Assembly assemblyToSearch)
             // Loop through all resources
             if (assemblyToSearch.FullName != null)
             {
-                // Get stream resource info
-                var streamResourceInfo = Application.GetResourceStream(new Uri(GifSource, UriKind.RelativeOrAbsolute));
-                if (streamResourceInfo != null)
+                try
                 {
-                    return (Bitmap) Image.FromStream(streamResourceInfo.Stream);
+                    // Get stream resource info
+                    var streamResourceInfo = Application.GetResourceStream(new Uri(GifSource, UriKind.RelativeOrAbsolute));
```

Comment:
Please check if GifSource == null. That saves us another first chance exception.